packages/common/src/components/ActionServiceDropdown/ActionServiceDropdown.tsx

```diff
@@ -85,7 +97,7 @@ const ActionsAsDropdown = ({
         .map(({ id, label, cta, disabled, disabledTooltip, tooltip }) => (
           <DropdownItem
             key={id}
-            onClick={typeof cta === 'function' ? cta : () => cta.href && history.push(cta.href)}
+            onClick={() => onClick(cta)}
```

Comment:
we need that actual onClick out, doing `() => onCLick` override the original onClick type
```suggestion
            onClick={onClick}
```